server.py

```diff
@@ -165,7 +165,10 @@ async def play(url: str):
         url_type = _get_url_type(url)
         # Check if the given url is a valid video or playlist
         if url_type == UrlType.VIDEO:
-            threading.Thread(target=handle_play, args=(url,)).start()
+            if loop:
+                threading.Thread(target=handle_play, args=(url,loop)).start()
+            else:
+                threading.Thread(target=handle_play, args=(url,False)).start()
```

Comment:
why have an if/else? we can just start the thread and pass in `loop`

Reply:
done